lib/preface_make/bounded_lattice.mli

```diff
@@ -0,0 +1,53 @@
+(** Building a {!module:Preface_specs.Bounded_lettice} *)
+
+(** {2 Using bounded join semilattice and bounded meet semilattice}
+
+    Build a {!module-type:Preface_specs.BOUNDED_LATTICE} using
+    {!module-type:Preface_specs.BOUNDED_LATTICE.WITH_BOUNDED_JOIN_AND_BOUNDED_MEET}. *)
+
+module Via_bounded_join_and_bounded_meet
+    (Req : Preface_specs.Bounded_lattice.WITH_BOUNDED_JOIN_AND_BOUNDED_MEET) :
+  Preface_specs.BOUNDED_LATTICE with type t = Req.t
+
+(** {1 Over Bounded join semilattice and over bounded meet semilattice}
+
+    Produces a {!module-type:Preface_specs.BOUNDED_LATTICE} from a
+    {!module-type:Preface_specs.BOUNDED_JOIN_SEMILATTICE}. and a
+    {!module-type:Preface_specs.BOUNDED_MEET_SEMILATTICE}. *)
+
+module Over_bounded_join_and_over_bounded_meet
```

Comment:
Even it is a bit verbose, I suggest to rename the module `Over_bounded_join_lattice_and_bounded_meet_lattice`.

Reply:
No problem, soi guess you want me to make that change everywhere i choose to write "bounded_x" instead of "bounded_x_lattice" ? (in specs etc)

Reply:
Yes please. I think this is important to have a clear separation between stuff that we can built on top of function (using via) and stuff that we can built on top of other abstraction (over)

Reply:
done